bpdm-common/src/main/kotlin/org/eclipse/tractusx/bpdm/common/dto/IBaseBusinessPartnerDto.kt

```diff
@@ -17,11 +17,10 @@
  * SPDX-License-Identifier: Apache-2.0
  ******************************************************************************/
 
-package org.eclipse.tractusx.bpdm.gate.api.model
+package org.eclipse.tractusx.bpdm.common.dto
 
 import io.swagger.v3.oas.annotations.media.ArraySchema
 import io.swagger.v3.oas.annotations.media.Schema
-import org.eclipse.tractusx.bpdm.common.dto.ClassificationDto
 import org.eclipse.tractusx.bpdm.common.dto.openapidescription.CommonDescription
 
```

Comment:
Contains externalId Field. I think the externalId should be only part of the Gate and not be put in the common

Reply:
Because there is no `externalId` in the Orchstrator model? Done!

Reply:
Yep, externalId is an identifier unique to the Gate. The orchestrator has cleaning task identifiers which the Gate uses to identify its own business partners